In-house patch to remove unnecessary dependencies from Horizon's
requirements files. The specific reasons are as follows:

eventlet	Not applicable

kombu		Not applicable

pyscss		Not applicable

--- horizon-2014.2.2/horizon.egg-info/requires.txt.~1~	2015-02-05 09:08:25.000000000 -0800
+++ horizon-2014.2.2/horizon.egg-info/requires.txt	2015-02-25 00:24:13.790331173 -0800
@@ -3,13 +3,10 @@ Django>=1.4.2,<1.7
 django_compressor>=1.4
 django_openstack_auth>=1.1.7,!=1.1.8
 django-pyscss>=1.0.3  # BSD License2 clause
-eventlet>=0.15.1,<0.16.0
 httplib2>=0.7.5
 iso8601>=0.1.9
-kombu>=2.5.0
 lockfile>=0.8
 netaddr>=0.7.12
-pyScss>=1.2.1,<1.3  # MIT License
 python-ceilometerclient>=1.0.6
 python-cinderclient>=1.1.0
 python-glanceclient>=0.14.0
@@ -40,4 +37,4 @@ xstatic-jquery-ui>=1.10.1  # MIT License
 xstatic-jsencrypt>=2.0.0.2  # MIT License
 xstatic-qunit>=1.14.0.2  # MIT License
 xstatic-rickshaw>=1.5.0  # BSD Licenseprior
-xstatic-spin>=1.2.5.2  # MIT License
+xstatic-spin>=1.2.5.2  # MIT License
\ No newline at end of file
--- horizon-2014.2.2/requirements.txt.~1~	2015-02-05 09:06:50.000000000 -0800
+++ horizon-2014.2.2/requirements.txt	2015-02-25 00:24:12.542805712 -0800
@@ -13,15 +13,12 @@ Django>=1.4.2,<1.7
 django_compressor>=1.4
 django_openstack_auth>=1.1.7,!=1.1.8
 django-pyscss>=1.0.3  # BSD License (2 clause)
-eventlet>=0.15.1,<0.16.0
 httplib2>=0.7.5
 iso8601>=0.1.9
-kombu>=2.5.0
 # Horizon Utility Requirements
 # for SECURE_KEY generation
 lockfile>=0.8
 netaddr>=0.7.12
-pyScss>=1.2.1,<1.3  # MIT License
 python-ceilometerclient>=1.0.6
 python-cinderclient>=1.1.0
 python-glanceclient>=0.14.0
